apps/console/src/features/users/components/guests/pages/invite-parent-org-user.tsx

```diff
@@ -281,7 +143,25 @@ export const InviteParentOrgUser: FunctionComponent<InviteParentOrgUserPropsInte
                                     </Typography>
                                 </Hint>
                             ) }
-                            component={ TextFieldAdapter }
+                            component={ AutocompleteFieldAdapter }
+                            options={ [] }
+                            renderTags={ (
+                                value: readonly string[],
+                                getTagProps: AutocompleteRenderGetTagProps
+                            ) =>
+                                value.map((option: string, index: number) => (
+                                    <Chip
+                                        key={ index }
+                                        size="small"
+                                        sx={ { marginLeft: 1 } }
+                                        className="oxygen-chip-beta"
```

Comment:
We don't need to explicitly specify oxygen classes here, do we?

cc:@savindi7

Reply:
Currently, we are going to have to add this class to get the styles of a beta label..